lib/telemetry.js

```diff
@@ -5,9 +5,13 @@ import { createHash } from 'node:crypto'
 import Sentry from '@sentry/node'
 import assert from 'node:assert'
 import { platform } from 'node:os'
+import fs from 'node:fs/promises'
+import { join } from 'node:path'
+import { paths } from './paths.js'
 
 const { FIL_WALLET_ADDRESS } = process.env
 
+const pkg = JSON.parse(await fs.readFile(join(paths.repoRoot, 'package.json')))
```

Comment:
Interesting. Node.js does not support importing JSON files yet? I vaguely remember import from ... followed by assert type json

Reply:
Unfortunately there's two issues with that:

```bash
$ nt

> @filecoin-station/core@2.1.0 test
> prettier --check . && standard && node--test

Checking formatting...
All matched files use Prettier code style!
standard: Use JavaScript Standard Style (https://standardjs.com)
  /Users/julian/dev/filecoin-station/core/lib/telemetry.js:8:35: Parsing error: Unexpected token assert (null)
$ ./bin/station.js
(node:56462) ExperimentalWarning: Importing JSON modules is an experimental feature. This feature could change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
```